google/cloud/storage/examples/storage_auth_samples.cc

```diff
@@ -57,6 +57,26 @@ void DefaultClient(std::vector<std::string> const& argv) {
   (argv.at(0), argv.at(1));
 }
 
+void ExplicitADCs(std::vector<std::string> const& argv) {
+  namespace examples = ::google::cloud::storage::examples;
+  if ((argv.size() == 1 && argv[0] == "--help") || argv.size() != 2) {
+    throw examples::Usage{
+        "explicit-adcs"
+        " <bucket-name> <object-name>"};
+  }
+  //! [explicit-adcs]
+  namespace gcs = google::cloud::storage;
```

Comment:
Fully qualify to match namespace alias a dew lines up?

Reply:
Done here and elsewhere on this file.